R/fit.R

```diff
@@ -28,6 +28,9 @@ CmdStanFit <- R6::R6Class(
     },
 
     draws = function(variables = NULL) {
+      if (!length(self$output_files(include_failed = FALSE))) {
+        stop("Fitting failed. Unable to retrieve the draws.")
```

Comment:
Can you add `call. = FALSE` to `stop()`? It makes for nicer error messages. (Same thing below but just commenting here)